### ISSUE TITLE
compatibility with plaidml backend

### ISSUE THREAD

**Issue by Payback80** [philipperemy/keras-tcn#17]
Hello i use plaidml as keras backend because it supports AMD graphics cards

there are two issues with your implementation
1) spatialdropout is not supported by plaidml so 
x = SpatialDropout1D(dropout_rate, name='spatial_dropout1d_%d_s%d_%f' % (i, s, dropout_rate))(x)
will not work 
https://github.com/plaidml/plaidml/issues/165

2) it seems that keras.backend.max isn't implemented too 

If you don't have time to fix it can you suggest a workaround?

**Comment by philipperemy** [comment]
@Payback80 yes sure.

1. Instead of SpatialDropout1D you can simply use Dropout. It will be slightly different but results should be the same.
2. We apply max() in the channel normalization. In that case you may use BatchNormalization!

**Comment by philipperemy** [comment]
Inactivity. Please re-open when you have something new!
